python_modules/dagster/dagster/_core/definitions/repository_definition/repository_definition.py

```diff
@@ -264,17 +265,21 @@ def asset_checks_defs_by_key(self) -> Mapping[AssetKey, "AssetChecksDefinition"]
         return self._repository_data.get_asset_checks_defs_by_key()
 
     def has_implicit_global_asset_job_def(self) -> bool:
+        """Returns true is there is a single implicit asset job for all asset keys in a repository."""
         return self.has_job(IMPLICIT_ASSET_JOB_NAME)
 
     def get_implicit_global_asset_job_def(self) -> JobDefinition:
-        return self.get_job(IMPLICIT_ASSET_JOB_NAME)
-
-    def get_implicit_asset_job_names(self) -> Sequence[str]:
-        return [IMPLICIT_ASSET_JOB_NAME]
+        """A useful conveninence method for repositories where there are a set of assets with
```

Comment:
Why did this change? Seems like we should be confident that `get_job(IMPLICIT_ASSET_JOB_NAME)` would return a job correctly, right? And if it didn't, then the cause wouldn't be multiple partitioning schemes, it'd be some other sort of weird framework error if I'm understanding correctly

Reply:
Oops I think this was some sort of merge conflict. You are correct.